test/github-issues/1576/entity/Category.ts

```diff
@@ -4,12 +4,12 @@ import {Column, PrimaryGeneratedColumn, Entity} from "../../../../src";
 export class Category {
 
     @PrimaryGeneratedColumn()
-    id: number|null;
+    id: number;
 
     @Column()
     name: string;
 
     @Column()
-    name2: string|null;
```

Comment:
Not sure why `|null` was added here(only in next branch). Reflect metadata thinks it's an object then and we guess wrong column type.